unstructured/ingest/evaluate.py

```diff
@@ -123,6 +132,73 @@ def measure_edit_distance(
     _display(agg_rows, headers)
 
 
+@main.command()
+@click.option("--output_dir", type=click.STRING, help="Directory to a structured output.")
+@click.option(
+    "--output_list",
+    type=click.STRING,
+    multiple=True,
+    help="Optional: list of selected structured output file names under the \
+        directory to be evaluate. If none, all files under directory will be use.",
+)
+@click.option("--source_dir", type=click.STRING, help="Directory to a structured source.")
+@click.option(
+    "--source_list",
+    type=click.STRING,
+    multiple=True,
+    help="Optional: list of selected structured source file names under the directory \
+        to be evaluate. If none, all files under directory will be use.",
+)
+@click.option(
+    "--export_dir",
+    type=click.STRING,
+    default="metrics",
+    help="Directory to save the output evaluation metrics to. Default to \
+        [your_working_dir]/metrics/",
+)
+def measure_element_type_accuracy(
+    output_dir: str,
+    output_list: Optional[List[str]],
+    source_dir: str,
+    source_list: Optional[List[str]],
+    export_dir: str,
+):
+    if not output_list:
+        output_list = _listdir_recursive(output_dir)
+    if not source_list:
+        source_list = _listdir_recursive(source_dir)
+
+    rows = []
+    accuracy_scores: List[float] = []
+
+    for doc in output_list:  # type: ignore
+        fn = (doc.split("/")[-1]).split(".json")[0]
+        connector = doc.split("/")[0]
+        if doc in source_list:  # type: ignore
+            output = get_element_type_frequency(_read_json(os.path.join(output_dir, doc)))
+            source = get_element_type_frequency(_read_json(os.path.join(source_dir, doc)))
```

Comment:
what are we currently using for gold standard element output, the expected_structured_output? From my understanding, the expected_structured_output represents the current state of the unstructured output for a given document, which might not necessarily be the gold standard human element division/classification. 

We will definitely need to start an odetta task to get these gold standard outputs, but in the meantime if it's only a few small documents maybe it should be done manually. By using expected_structured_output, the element accuracy result will always be 100% because by construction the expected_structured_output has to match the unstructured output exactly